app/views/total_income_tax_includes/ScottishTableView.scala.html

```diff
@@ -50,8 +50,8 @@
     <table>
         <thead>
             <tr>
-                <th scope="col">@messages("ats.total_income_tax.scottish_income_tax.table.total")</th>
-                <th>@Html(viewUtils.toCurrency(taxModel.scottishTotalTax))</th>
+                <th scope="row">@messages("ats.total_income_tax.scottish_income_tax.table.total")</th>
+                <td class="align-right">@Html(viewUtils.toCurrency(taxModel.scottishTotalTax))</td>
```

Comment:
Could you use 'numeric' class? I believe it has  `text-align: right;` built in...
https://govuk-elements.herokuapp.com/data/#data-table-numeric

Reply:
good shout :thumbsup: